cmd/trace-agent/test/testsuite/config_set_test.go

```diff
@@ -0,0 +1,53 @@
+package testsuite
+
+import (
```

Comment:
Please go ahead and group the imports into 3 groups like everywhere else (std lib, internal, external)